ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V100_1__Append_only_cleanup.sql

```diff
@@ -0,0 +1,186 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
this migration is quite lightweight compared to the others, but makes sense from logical composition point of view

Reply:
It is indeed. It's less about reporting progress but more about having the possibility to compare old and new data before the old data is deleted.